inc/form.class.php

```diff
@@ -821,7 +827,8 @@ public static function getFormList(int $rootCategory = 0, string $keywords = '',
                'description'      => '',
                'type'             => 'faq',
                'usage_count'      => $faq['view'],
-               'is_default'       => false
+               'is_default'       => false,
+               'tile_template'         => 'fix'
```

Comment:
```suggestion
               'tile_template'    => 'fix'
```

The value 'fix' does not matches any of your new constants.